frontend/src/pages/projects/screens/detail/ProjectDetailsSidebar.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import {
+  JumpLinks,
+  JumpLinksItem,
+  Sidebar,
+  SidebarContent,
+  SidebarPanel,
+} from '@patternfly/react-core';
+
+const ProjectDetailsSidebarWrapper: React.FC = ({ children }) => {
```

Comment:
```suggestion
const ProjectDetailsSidebar: React.FC = ({ children }) => {
```
Components should share the name of the file